smac/facade/smac_facade.py

```diff
@@ -173,7 +173,7 @@ def __init__(self,
         # initialize optimizer on acquisition function
         if acquisition_function_optimizer is None:
             acquisition_function_optimizer = InterleavedLocalAndRandomSearch(
-                acquisition_function, scenario.cs
+                acquisition_function, scenario.cs, np.random.RandomState(seed=rng.randint(MAXINT))
```

Comment:
why do we use MAXINT as seed here? I would expect a user-given seed or rng here.

Reply:
The rng is initialized with a user-given seed and so the seed generated by it is also user-dependent.

Reply:
1234567890 was used previously, didn't change it during the refactoring. Now using MAXINT.